packages/firestore/src/lite/database.ts

```diff
@@ -134,6 +134,13 @@ export class FirebaseFirestore implements FirestoreService {
     return this._terminateTask;
   }
 
+  toJSON(): object {
+    return {
+      databaseId: this._databaseId,
```

Comment:
Should we include the app as well? I think the developers might want to see the app name here, since it acts as the main identifier for all Firebase components.